src/accessors/IPersistence.ts

```diff
@@ -0,0 +1,28 @@
+import { IPersistenceRead } from './IPersistenceRead';
+/**
+ * An instance of this interface will be constructed by the Rocketlet Manager
+ * which allows the Rocketlet to access its own (generic) persistence.
+ * In order to provide  high-performance access, the associated Rocket.Chat
+ * model provided in `Rocketlet.getRocketChatAssociation()` is considered when
+ * creating the persistence-accessor
+ */
+export interface IPersistence extends IPersistenceRead{
+    /**
+     * Adds a new record representing the state of the Rocketlet
+     * @param referenceId The ID of the instance of the associated model
+     * @param data The Rocketlets' very own data
+     *
+     * @return An ID of the Rocketlet data
+     */
+    create(referenceId: string, data: object): string;
```

Comment:
Can you explain this? Not sure I follow the referenceId purpose and the associating a Rocketlet's custom data to a model inside Rocket.Chat..